docs/sources/setup-grafana/configure-security/configure-authentication/generic-oauth/index.md

```diff
@@ -380,15 +267,18 @@ role_attribute_path = contains(info.roles[*], 'admin') && 'GrafanaAdmin' || cont
 
 ## Team synchronization
```

Comment:
Why is this here? Can this information be included in the team sync setting of the table above?